### PR TITLE
Improve size for app adaptive scaffold

### PR DESCRIPTION
**Pull request by nqhhdev**
- Updated every styles file for upscaling
- Removed unused styles 

image

image

image

image

#1065

### CODE REVIEW

File: lib/pages/chat_list/client_chooser_button_style.dart [hunk 1]
```diff
@@ -1,6 +1,8 @@
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
 class ClientChooserButtonStyle {
-  static double get avatarSizeInAppBar => 40.0;
-  static double get avatarFontSizeInAppBar => 17.0;
+  static double get avatarSizeInAppBar => 40.0.r;
```
Comment (hoangdat) [inline lib/pages/chat_list/client_chooser_button_style.dart:4]: why it is `r`?

File: lib/config/themes.dart [hunk 2]
```diff
@@ -90,13 +104,13 @@ abstract class TwakeThemes {
             : Colors.blueGrey.shade900,
         popupMenuTheme: PopupMenuThemeData(
           shape: RoundedRectangleBorder(
-            borderRadius: BorderRadius.circular(AppConfig.borderRadius),
+            borderRadius: BorderRadius.circular(AppConfig.borderRadius.r.r),
```
Comment (hoangdat) [inline lib/config/themes.dart:107]: why need to use double `.r`?

File: lib/pages/chat_list/client_chooser_button_style.dart (hunk 1 above)
Comment (imGok) [reply lib/pages/chat_list/client_chooser_button_style.dart]: Fixed

**Comment by nqhhdev** [comment]
Pls rebase main before doing it, because we updated some thing about style text, height, width of appbar

**Comment by imGok** [comment]
@hoangdat size of widget looks good overall except fontsize, I'll check manually why they are not scaling properly

**Comment by hoangdat** [comment]
in 1280 x 800 resolution, we have this problem

image

**Comment by hoangdat** [comment]
- [ ] avatar also be smaller in 1280 x 800

**Pull request closed without merging**